### PR TITLE
Paramiko support

### PR DESCRIPTION
**Pull request by blamarvt**

### CODE REVIEW

File: stacktester/config.py [hunk 1]
```diff
@@ -39,6 +39,11 @@ class NovaConfig(object):
         """API key to use when authenticating. Defaults to 'admin_key'."""
         return self.get("api_key", "admin_key")
 
+    @property
+    def ssh_username(self):
```
Comment (markwash) [inline stacktester/config.py:43]: Could you add a similar config option for ssh port?

**Comment by throughnothing** [comment]
This looks good to me once we can configure ssh port as well.

**Comment by blamarvt** [comment]
I thought about doing that...but then I thought: "Who the heck changes
the ssh port?" ;)

I'll add support in the morning, shouldn't take but a second, should I
add support for SSH password too? Right now it just assumes you have
SSH keys set up.

On Thu, Jun 23, 2011 at 9:56 PM, throughnothing
reply@reply.github.com
wrote:

**Comment by markwash** [comment]
I am good with just ssh keys.

On Thu, Jun 23, 2011 at 11:44 PM, blamarvt
reply@reply.github.com
wrote:

**Comment by blamarvt** [comment]
Please review, I have added support for changing the SSH port.

**Pull request merged**